actionview/lib/action_view/helpers/tags/translator.rb

```diff
@@ -10,26 +10,24 @@ def initialize(object, object_name, method_and_value, scope:)
         end
 
         def translate
-          translated_attribute = I18n.t("#{object_name}.#{method_and_value}", default: i18n_default, scope: scope).presence
+          translated_attribute = I18n.t("#{@object_name}.#{@method_and_value}", default: i18n_default, scope: @scope).presence
           translated_attribute || human_attribute_name
         end
 
         private
 
-        attr_reader :object_name, :method_and_value, :scope, :model
```

Comment:
According to the warning this was the line that gave the warning.

I don't understand the error message (private attribute?), but perhaps the issue is with some of the definitions here.
Could you try putting each of them in separate declarations and post how many lines give warnings?